pkg/api/query/querypb/query.proto

```diff
@@ -64,6 +69,8 @@ message QueryResponse {
     /// timeseries is one series from the result of the executed query.
     prometheus_copy.TimeSeries timeseries = 2;
   }
+
+  QueryStats stats = 3;
```

Comment:
Won't this cause stats to be sent for each series? I think we want to move stats into the `oneof` and send it at the end once all series have been sent.

Reply:
Good catch. Changed